pkg/plugin/datasource.go

```diff
@@ -206,6 +208,11 @@ func (d *Datasource) CheckHealth(ctx context.Context, req *backend.CheckHealthRe
 		res.Message = "'Host' cannot be empty"
 		return res, nil
 	}
+	if len(config.Username) == 0 {
+		res.Status = backend.HealthStatusError
+		res.Message = "'Username' cannot be empty"
```

Comment:
To connect to Timeplus Proton, the user doesn't need to set username/password. Maybe use 'default' if the username is not set

Reply:
updated